manon/components/checkbox.scss

```diff
@@ -2,41 +2,25 @@
 @use "../variables" as theme;
 @use "../functions/validators";
 
-.checkbox {
-  padding: 0;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: nowrap;
-
-  @include theming.apply-from-theme(
-    (
-      gap: theme.$checkbox-gap,
-      align-items: theme.$checkbox-align-items,
-    )
-  );
-
+@mixin checkbox-input {
   input[type="checkbox"] {
     padding: 0;
+    margin: 0;
     cursor: pointer;
-
-    @include theming.apply-from-theme(
-      (
-        width: theme.$checkbox-width,
-        min-width: theme.$checkbox-width,
-        height: theme.$checkbox-height,
-        min-height: theme.$checkbox-height,
-        accent-color: theme.$checkbox-accent-color,
-      )
-    );
+    border: 1px solid #ced4da;
```

Comment:
We want the theme builders to be able to set any visual styling. We also want to be as least opinionated with base styling as we can be. Add it as a variable instead ideally without a base value. (default can be set to null)